rust/private/rustc.bzl

```
@@ -492,7 +492,9 @@ def construct_arguments(
         if rust_common.crate_info in data:
             dep_crate_info = data[rust_common.crate_info]
             if dep_crate_info.type == "bin":
-                env["CARGO_BIN_EXE_" + dep_crate_info.output.basename] = dep_crate_info.output.short_path
+                # Trying to make CARGO_BIN_EXE_{} canonical across platform by strip out extension if exists
+                env_basename = dep_crate_info.output.basename[:-(1 + len(dep_crate_info.output.extension))] if len(dep_crate_info.output.extension) > 0 else dep_crate_info.output.basename
+                env["CARGO_BIN_EXE_" + env_basename] = "${{pwd}}/{}".format(dep_crate_info.output.short_path)
```

I think this needs to be `output.path` since `${pwd}` points to the exec root
```suggestion
                env["CARGO_BIN_EXE_" + env_basename] = "${{pwd}}/{}".format(dep_crate_info.output.path)
```